AppBuilder/platform/views/viewComponent/ABViewDetailComponent.js

```diff
@@ -7,7 +7,21 @@ module.exports = class ABViewDetailComponent extends ABViewContainerComponent {
    }
 
    ui() {
-      return super.ui();
+      let _ui = super.ui();
+
+      // this wrapper allows the detail view to have a
+      // card appearance as well as enables the edit and
+      // details functions to work when clicked
+      return {
+         id: `ABViewDetail_${this.view.id}`,
```

Comment:
We might need to update this `.id` value because `super.ui()` should contain this value already.